plugins/formal-verification/formver.common/src/org/jetbrains/kotlin/formver/UnsupportedFeatureBehaviour.java

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
Why are we defining a Java file?

Reply:
Having a static method here is convenient (same as in `LogLevel`).

Reply:
I think we can achieve the same result in a Kotlin file 🤔

```kotlin
enum class UnsupportedFeatureBehaviour {
    ASSUME_UNREACHABLE,
    THROW_EXCEPTION;

    companion object {
        @JvmStatic
        fun defaultBehaviour(): UnsupportedFeatureBehaviour = THROW_EXCEPTION
    }
}
```

Reply:
Is there any benefit to doing so?